DEVELOPING.md

```diff
@@ -184,5 +184,5 @@ following incantations.
 
     > ++SCALA_VERSION
     > ;compiler/publishLocal;library/publishLocal;testInterface/publishLocal;testBridge/publishLocal;jUnitRuntime/publishLocal;jUnitPlugin/publishLocal
-    > ++2.10.7
+    > ++2.12.8
```

Comment:
2.12.8 is the default, so that shouldn't even be necessary.

Reply:
Hum, but there is a `++SCALA_VERSION` before it. Or did you want me to change the order?

Reply:
Oh right. I missed that. No never mind, then.